railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults.rb.tt

```diff
@@ -1,37 +1,36 @@
 # Be sure to restart your server when you modify this file.
-# This file contains all the new default configuration options from
-# Rails 5.0.
-<%- unless options[:skip_active_record] -%>
-
-# Require `belongs_to` associations by default. This is a new Rails 5.0
-# default, so it is introduced as a configuration option to ensure that apps
-# made on earlier versions of Rails are not affected when upgrading.
-Rails.application.config.active_record.belongs_to_required_by_default = true
-<%- end -%>
-
-# Do not halt callback chains when a callback returns false. This is a new
-# Rails 5.0 default, so it is introduced as a configuration option to ensure
-# that apps made with earlier versions of Rails are not affected when upgrading.
-ActiveSupport.halt_callback_chains_on_return_false = false
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+# They will be removed in the next major Rails version.
+#
+# Flip the defaults to the value of past versions then once upgraded
+# begin migrating to the new default one by one.
+#
+# Read the Rails 5.0 release notes for more info on each option.
 <%- unless options[:api] -%>
 
-# Enable per-form CSRF tokens.
+# Enable per-form CSRF tokens. This is new feature of Rails 5.
 Rails.application.config.action_controller.per_form_csrf_tokens = true
 
-# Enable origin-checking CSRF mitigation.
+# Enable origin-checking CSRF mitigation. This is new feature of Rails 5.
```

Comment:
True these are new features but enabling them when upgrading has ramifications on apps. Let's wrap in update and set to false on upgrading apps; new apps get these features enabled.

Then for consistency, let's add `Previous versions had false`.